### PR TITLE
feat(i18n): add Arabic(AR) translation

### PR DESCRIPTION
**Pull request by OthmanAliModaes**
Adding **Arabic** language.

## Brief Information

This pull request is in the type of:
- Add new language.

### What does this PR do?
- Add **Arabic** language file: **src/i18n/langAR.ts**

### Other information
**- in progress...**

### CODE REVIEW

**Review by Ovilia — CHANGES_REQUESTED** [review]
Thanks for contribution. Why are most of them not translated?

**Comment by OthmanAliModaes** [comment]
I will translate the rest in a few hours.

File: src/i18n/langAR.ts [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+    time: {
+        month: [
+            'يناير', 'فبراير', 'مارس', 'أبريل', 'مايو', 'يونيو',
+            'يوليو', 'أغسطس', 'سبتمبر', 'أكتوبر', 'نوفمبر', 'ديسمبر'
+        ],
+        monthAbbr: [
+            'يناير', 'فبراير', 'مارس', 'أبريل', 'مايو', 'يونيو',
+            'يوليو', 'أغسطس', 'سبتمبر', 'أكتوبر', 'نوفمبر', 'ديسمبر'
+        ],
```
Comment (plainheart) [inline src/i18n/langAR.ts:34]: It looks like the abbreviation is the same as the full form. Besides, the month items seem not to be sorted from 1~12 and the dayOfWeek items seem not to be sorted from Sunday-Saturday.

Ref: https://github.com/amcharts/amcharts5/blob/master/src/locales/ar.ts#L227C14-L266

**Review by plainheart — CHANGES_REQUESTED** [review]

File: src/i18n/langAR.ts (hunk 1 above)
Comment (OthmanAliModaes) [reply src/i18n/langAR.ts:34]: The problem is not in the code, but in the code editor, as it displays texts that contain RTL languages ​​in reverse.
I suffer from this problem a lot, as an Arab software developer.
The code display is wrong, but the output is correct.
The same applies to the order of days.

EX:

    var monthAbb = [
        'يناير', 'فبراير', 'مارس', 'أبريل', 'مايو', 'يونيو',
        'يوليو', 'أغسطس', 'سبتمبر', 'أكتوبر', 'نوفمبر', 'ديسمبر'
    ]

    console.log(monthAbb[0]) // يناير = January
    console.log(monthAbb[11]) // ديسمبر = December

    var dayOfWeek = [
        'الأحد', 'الإثنين', 'الثلاثاء', 'الأربعاء', 'الخميس', 'الجمعة', 'السبت'
    ]
    console.log(dayOfWeek[0]) // الأحد = Sunday
    console.log(dayOfWeek[6]) // السبت = Saturday

Comment (plainheart) [reply src/i18n/langAR.ts:34]: Thanks for explanation. Get it.

**Review by plainheart — APPROVED** [review]

**Pull request merged**